sr_grasp/scripts/quick_grasp

```diff
@@ -59,6 +60,15 @@ class QuickGraspNode(object):
         else:
             rospy.logerr("Fail!")
 
+    def new_grasp(self, name):
+        grasp = Grasp()
+        grasp.id = name
+        positions = self.hand.read_all_current_positions()
+        for k, v in positions.iteritems():
+            positions[k] = radians(v)
```

Comment:
It is all in radians in the yaml! The shadow hand lib does degrees but that is a bad habit ;-)